analyzers/src/ITs.JsonParser/IssueParser.cs

```diff
@@ -47,10 +47,17 @@ public static void Execute(string inputPath, string outputPath)
         ConsoleHelper.WriteLineColor("Successfully wrote all Rule reports.", ConsoleColor.Green);
     }
 
-    private static InputReport[] Read(string rootPath) =>
-        Directory.GetFiles(rootPath, "*.json", SearchOption.AllDirectories)
-            .Select(x => new InputReport(x, SerializerOptions))
-            .ToArray();
+    private static InputReport[] Read(string rootPath)
+    {
+        var projects = Directory.GetDirectories(rootPath);
+        var inputReports = new List<InputReport>();
+        foreach (var project in projects)
+        {
+            inputReports.AddRange(Directory.GetFiles(Path.Combine(rootPath, project, "issues"))
```

Comment:
The `Path.Combine` with `rootPath` is weird here. `Directory.GetDirectories` is returning absolute full paths